### DISCUSSION TITLE
How know if VS display shows dashes

### DISCUSSION THREAD

**Discussion by JensKn** [flybywiresim/aircraft#4944] [Help] [answered]
I am able to read VS and FPA values, but I could not figure out how to determine if the VS display in the FCU shows dashes ("-----").

Any help appreciated!

Regards,
Jens

**Comment by aguther** [comment, marked as answer]
There is currently no way to read this info.

**Reply by JensKn** [reply]
Hello Andreas,

thanks for your reply! 

May I suggest to implement this and other information/behaviour which when missing prevents home cvockpit builders from using their hardware for the A32NX?

Do you know if the A32NX project addresses SimConnect or WASM gauge connectivity features (both implementaion and testing)?

Regards,
Jens

PS @aguther: I could have written this in German but prefered keeping my answer in English to allow more people to participate.
